### PR TITLE
Add data connector example

### PR DESCRIPTION
**Pull request by blackburn32**

### CODE REVIEW

**Review by Sheyne — COMMENTED** [review]
I think we should not commit this bin folder

**Review by Sheyne — COMMENTED** [review]
Nit: Do you like this file name being camel case?

File: data-connector-example/dataconnectors/data-connector-1/actions/import.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import {DataConnectorAsynchronousAction, SerializedFields} from 'lucid-extension-sdk';
+import { ApiFolder, convertToFolder, folderSchema } from '../collections/folders';
```
Comment (Sheyne) [inline data-connector-example/dataconnectors/data-connector-1/actions/import.ts]: Nit: different formatting styles for your imports

**Comment by blackburn32** [comment]
> I think we should not commit this bin folder

Removed, good callout

File: data-connector-example/dataconnectors/data-connector-1/actions/import.ts (hunk 1 above)
Comment (blackburn32) [reply data-connector-example/dataconnectors/data-connector-1/actions/import.ts]: Added prettier so these are consistent

**Comment by blackburn32** [comment]
> Nit: Do you like this file name being camel case?

Renamed

**Review by cball18 — APPROVED** [review]

**Review by jselvaraaj — APPROVED** [review]

**Pull request merged**